### ISSUE TITLE
text rendered at wrong size

### ISSUE THREAD

**Issue by rghvdberg** [michaelwillis/dragonfly-reverb#6]
see #5

**Comment by michaelwillis** [comment]
I think the problem is that the code is hardwired to look for the font in a specific place:

```
  fNanoFont  = fNanoText.createFontFromFile ( "lcd_solid","/usr/share/fonts/truetype/noto/NotoSans-Regular.ttf" );
```

Not everybody is going to have the Noto Sans font there, in which case I think it defaults to some other font. We would probably have the best luck making it use a font that gets compiled in. It looks like DPF already compiles in Deja Vu Sans, so we could try that.

**Comment by rghvdberg** [comment]
Gonna try moving the font in the source tree and make a relative path to the font. I tried that before on the old UI, but failed. Then I got lazy and just used a full path.

**Comment by michaelwillis** [comment]
If all else fails, you can use Deja Vu Sans the same way that Distrho MVerb does it: https://github.com/DISTRHO/MVerb/blob/master/plugins/MVerb/DistrhoUIMVerb.cpp#L36

**Comment by rghvdberg** [comment]
Yup, I know.
I'll look into it tomorrow.

**Comment by rghvdberg** [comment]
Found it:
Indeed dpf falls back to Deja Vu Sans when it can't find the font. (I wish it would just give a compile error :-/ )
I don't think a relative path to font will work , so we need to have it compiled in.
The Deja Vu Sans font is here -> `dragonfly-reverb/dpf/dgl/src/Resources.hpp`
@falkTX seem to have used a util to auto-generate this file and we need that one.

**Comment by rghvdberg** [comment]
latest PR should fix this :-)
please confirm

**Comment by michaelwillis** [comment]
Fixed by d63efd36af0cd535cde3cd3e1d838e0b9e802f52
